.circleci/config.yml

```diff
@@ -37,10 +37,16 @@ jobs:
       - run:
           name: Run tests with coverage check
           command: npm run test:coverage
+      - run:
```

Comment:
General note, what do you think about unifying the jobs the one? Because you lose the benefit of multi job when you cannot do them in parallel

Reply:
I get your point. We can have either:
- 1 big job => simple, quite fine
- 3 jobs: prepare (install deps) + build + test (both latest running in parallel) => it'll provide better UX I think, but maybe it's to complicated

What do you think?

Reply:
and a third option install+build and lint in parallel.
Also, the other two will probably be faster, maybe only the last one because persistent to workspace is not very fast

Reply:
OK, I'll go with the simple, single job option.